dbt/adapters/databricks/impl.py

```diff
@@ -67,3 +80,113 @@ def execute(
         finally:
             if staging_table is not None:
                 self.drop_relation(staging_table)
+
+    def list_relations_without_caching(
```

Comment:
It would be nice to add a new method in dbt-spark that returns the catalog field to avoid duplicate code in the future (but we need to know the release schedule for dbt-spark).

Reply:
Yes, we can ask `dbt-spark` to make the change.
Let's have them here and work on it in a separate PR for now.